az-core/src/main/java/azkaban/Constants.java

```diff
@@ -89,6 +89,8 @@
   // The flow exec id for a flow trigger instance unable to trigger a flow yet
   public static final int FAILED_EXEC_ID = -2;
 
+  // Name of the file which keeps project directory size
```

Comment:
Since this constant is an implementation detail rather than a user-facing API, is it better to define it in a place where it is used?

Reply:
according to the doc of this class, ```Constants used in configuration files or shared among classes``` the variable it could keep doesn't necessarily need to be a user-facing API.  
Even if the variable is defined in the class where it's used, associated test class  still needs to access it.

Reply:
It's generally a good practice to reduce the scope of a concept.  For example, if a constant is only used in a class, define it for that scope. The test classes can access it via FooClass.BAR if needed. 

We don't have a good way to organize our user-facing key names etc yet, the current arrangement for this central class is a compromise until we find a better solution. 

Make sense?

Reply:
updated, thanks.